tarxx-example.cpp

```diff
@@ -23,48 +23,191 @@
 // OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-
 #include "tarxx.h"
+#include <filesystem>
+#include <getopt.h>
```

Comment:
this needs to go into the `#ifdef __linux` block below.

you might need to find and link a library on some systems. not sure, though.

Reply:
should be part of glibc?